routes/api/comments/index.js

```diff
@@ -10,8 +26,23 @@ router.get('/:comment_id', (req, res) => {
   res.send('Read a comment');
 });
 
-router.post('/', (req, res) => {
-  res.send('Write a comment');
+router.post('/', (req, res, next) => {
+  let comment  = new Comment({
+    body: req.query.comment,
+    author_id: req.query.author_id,
+    asset_id: req.query.asset_id,
+    parent_id: req.query.parent_id,
+    status: req.query.status
+  });
+  comment.save(function(err, comment) {
+    if(err) {
+      res.status(500);
+      return next(err);
+    }
+    res.status(201);
+    res.send(comment.id);
+    next();
+  });
```

Comment:
This block could be 
```
comment.save().then(({id}) => {
  res.status(201).send(id);
}).catch(error => {
  next(error);
});
```

I'd like to avoid callbacks if possible, I feel like it makes the code a bit easier to reason about. I still have nightmares about my old js with a million `if (err) { // something }`